test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb

```diff
@@ -63,3 +63,8 @@
 describe command('berks version') do
   its(:exit_status) { should eq 0 }
 end
+
+describe file('/home/alfred/.git-credentials') do
+  it { should be_a_file }
+  its(:content) { should match('https://osuosl-manatee') }
```

Comment:
I would check for the whole line including the token to be sure it works properly.